src/ui/map.js

```diff
@@ -1025,7 +1025,11 @@ class Map extends Camera {
      * const projection = map.getProjection();
      */
     getProjection() {
-        return this.transform.getProjection();
+        const proj = this.transform.getProjection();
+        if (this._transitionFromGlobe) {
+            proj.name = "globe";
+        }
```

Comment:
@karimnaaji I address the issue here in a superficial (hacky?) way. An alternative would be to change the internal behavior of globe to be more consistent with other projections. (Preserve "globe" as `map._runTimeProjection` and get rid of `map._transitionFromGlobe`) What do you think is the best path forward here?

Reply:
>(Preserve "globe" as map._runTimeProjection and get rid of map._transitionFromGlobe) What do you think is the best path forward here?

Could you store the implicit and explicit projection in different members so that you would just return it here instead of overriding? `_runTimeProjection` would be renamed `_explicitProjection` as the naming doesn't reflect anything _runtime_, and you would always return the `_explicitProjection` here but use the `_implicitProjection` internally whenever it is set.

`_transitionFromGlobe` could eventually be removed as it may reduce to checks against whether the implicit projection is globe or mercator.